portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,124 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.Entity;
+
+/** Comment data class */
+public class Comment {
+
+  /* Comment text */
+  private String commentText;
+
+  /* Comment sentiment */
```

Comment:
```suggestion
  /* Whether a sentiment is associated with the comment text */
```

Reply:
I delete the hasSentimentScore flag anyway (to address Jake's comment), so this comment will no longer apply.